### ISSUE TITLE
FontAwesome: Could not find icon with iconName=whatsapp and prefix=fab.

### ISSUE THREAD

**Issue by andreidiaconescu** [MurhafSousli/ngx-sharebuttons#418]
#### I am submitting a

- [x] Bug Report

#### What is the expected behavior?
I should see the whatsapp button with icon

#### What is the current behavior?
I see only the whatsapp button with no icon, and an error in the browser console:
`FontAwesome: Could not find icon with iconName=whatsapp and prefix=fab. This warning will become a hard error in 0.6.0.`

**UPDATE**:
for facebook button i get similar error:
`FontAwesome: Could not find icon with iconName=facebook-f and prefix=fab. This warning will become a hard error in 0.6.0.`

#### What are the steps to reproduce?

I use the "Share Button Component" as described at: https://github.com/MurhafSousli/ngx-sharebuttons/wiki/Share-Button-Component;
I have in my component view a piece of html like this
`<share-button button="whatsapp"></share-button>`
When the component shows, i only see the green button, but it is empty, with no icon.

#### Which versions are you using for the following packages?

Angular: 8.2.11
Angular CDK:8.2.3
Angular CLI:8.3.13
Angular-fontawesome:0.5.0
Share buttons: "@ngx-share/core": "7.1.4",  "@ngx-share/button": "7.1.4",

**Comment by nolafs** [comment]
Hey,

You need to import the icons

Create a icons.ts with the following code:

```
import { library } from '@fortawesome/fontawesome-svg-core';

import { faFacebookF } from '@fortawesome/free-brands-svg-icons/faFacebookF';
import { faTwitter } from '@fortawesome/free-brands-svg-icons/faTwitter';
import { faRedditAlien } from '@fortawesome/free-brands-svg-icons/faRedditAlien';
import { faLinkedinIn } from '@fortawesome/free-brands-svg-icons/faLinkedinIn';
import { faGooglePlusG } from '@fortawesome/free-brands-svg-icons/faGooglePlusG';
import { faTumblr } from '@fortawesome/free-brands-svg-icons/faTumblr';
import { faPinterestP } from '@fortawesome/free-brands-svg-icons/faPinterestP';
import { faWhatsapp } from '@fortawesome/free-brands-svg-icons/faWhatsapp';
import { faVk } from '@fortawesome/free-brands-svg-icons/faVk';
import { faFacebookMessenger } from '@fortawesome/free-brands-svg-icons/faFacebookMessenger';
import { faTelegramPlane } from '@fortawesome/free-brands-svg-icons/faTelegramPlane';
import { faMix } from '@fortawesome/free-brands-svg-icons/faMix';
import { faXing } from '@fortawesome/free-brands-svg-icons/faXing';
import { faLine } from '@fortawesome/free-brands-svg-icons/faLine';

import { faCommentAlt } from '@fortawesome/free-solid-svg-icons/faCommentAlt';
import { faMinus } from '@fortawesome/free-solid-svg-icons/faMinus';
import { faEllipsisH } from '@fortawesome/free-solid-svg-icons/faEllipsisH';
import { faLink } from '@fortawesome/free-solid-svg-icons/faLink';
import { faExclamation } from '@fortawesome/free-solid-svg-icons/faExclamation';
import { faPrint } from '@fortawesome/free-solid-svg-icons/faPrint';
import { faCheck } from '@fortawesome/free-solid-svg-icons/faCheck';
import { faEnvelope } from '@fortawesome/free-solid-svg-icons/faEnvelope';
import {faBook, faCoffee, faInfo, faLightbulb} from '@fortawesome/free-solid-svg-icons';

const icons = [
  faFacebookF, faTwitter, faLinkedinIn, faGooglePlusG, faPinterestP, faRedditAlien, faTumblr,
  faWhatsapp, faVk, faFacebookMessenger, faTelegramPlane, faMix, faXing, faCommentAlt, faBook, faLine,
  faEnvelope, faCheck, faPrint, faExclamation, faLink, faEllipsisH, faMinus, faLightbulb, faCoffee, faInfo
];

library.add(...icons);

```

Then bootstrap it to your application.

**Comment by eliavmaman** [comment]
I've import icons.ts  but its still not working

**Comment by vanessasoutoc** [comment]
Solution??

**Comment by nolafs** [comment]
Did you added bootstrap to your app?

```
import { enableProdMode } from '@angular/core';
import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';

import { AppModule } from './app/app.module';
import { environment } from './environments/environment';

import { hmrBootstrap } from './hmr';
import './icons';

if (environment.production) {
  enableProdMode();
}
....
```

**Comment by vanessasoutoc** [comment]
@nolafs is Ionic App. :/

**Comment by nolafs** [comment]
Should be fine, you just need import it at some point.

**Comment by vanessasoutoc** [comment]
@nolafs Thank you! It's work. Very Good!!!

**Comment by SyrianProfessional** [comment]
https://murhafsousli.github.io/ngx-sharebuttons/#/icons

see here and do what said

you can make a file in src to include the icons and after that import into main.ts

**Comment by lalithsprasad** [comment]
> https://murhafsousli.github.io/ngx-sharebuttons/#/icons
> 
> see here and do what said
> 
> you can make a file in src to include the icons and after that import into main.ts

This also does not work for me,  i am using Angular 8. so any answers?

**Comment by MurhafSousli** [comment]
Please try the latest beta version `v8.0.0-beta.1`, details is available in #432

**Comment by lalithsprasad** [comment]
okay thank you Murhaf :)
